onnxscript/rewriter/ort_fusions/_core.py

```diff
@@ -53,6 +54,7 @@ def fuse_xformers(model: ir.Model) -> ir.Model:
     fuse_cos_sin_cache(model)
     fuse_sdpa(model)
     fuse_mha(model)
+    fuse_attention(model)
```

Comment:
This is fine for now. At some point, we should consider a potential optimization: typically we expect a MHA or Attention or GQA etc. ... so if one of them applies, the others may not be necessary to apply.

Reply:
That makes sense, although won't the Attention op expect the MHA fusion to be done beforehand?

Reply:
Right ... so, it will need to be a bit more complicated. If MHA doesn't succeed, Attention need not be tried, but GQA can be tried, etc.

Reply:
Will add this as a follow up PR once this is merged.